utilities/resourceRemoval/Remove-DeployedModule.ps1

```diff
@@ -100,7 +79,16 @@ function Remove-DeployedModule {
             }
         } else {
             Write-Verbose 'Handle resource group level removal'
-            $resourcesToRemove = Get-AzResource -Tag @{ removeModule = $moduleName } -ResourceGroupName $resourceGroupName
+
+            $retryCount = 1
+            $retryLimit = 40
+            $retryWaitInSeconds = 15
```

Comment:
Repeated code? Again, take these from parameters in my opinion.

Reply:
done